Documentation/guides.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<guides xmlns="https://www.phpdoc.org/guides" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://www.phpdoc.org/guides ../vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd" links-are-relative="true">
+  <extension class="\T3Docs\Typo3DocsTheme\DependencyInjection\Typo3DocsThemeExtension" project-home="https://extensions.typo3.org/extension/typo3_console" project-contact="https://typo3.slack.com/archives/C07NARQGY" project-repository="https://github.com/TYPO3-Console/TYPO3-Console" project-issues="https://github.com/TYPO3-Console/TYPO3-Console/issues" edit-on-github-branch="main" edit-on-github="TYPO3-Console/TYPO3-Console" interlink-shortcode="helhum/typo3-console" typo3-core-preferred="stable"/>
+  <project title="TYPO3 Console" release="8.1.1" version="8.1.1" copyright="since 2014 by Helmut Hummel &amp; Contributors"/>
```

Comment:
Please format this for readability. See e.g. [georgringer/news](https://github.com/georgringer/news/blob/1a996aeb797a2bc7a5aa541e4108badc2f452548/Documentation/guides.xml)